src/DssSpell.t.sol

```diff
@@ -6,22 +6,11 @@ import "./DssSpell.t.base.sol";
 
 contract DssSpellTest is DssSpellTestBase {
 
-    
-    address constant COM_WALLET     = 0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9;
-    address constant FLIP_FLOP_FLAP         = 0x688d508f3a6B0a377e266405A1583B3316f9A2B3;
-    address constant FEEDBLACK_LOOPS    = 0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1;
-    address constant ULTRA_SCHUPPI         = 0x89C5d54C979f682F40b73a9FC39F338C88B434c6;
-    address constant FIELD_TECHNOLOGIES_INC = 0x0988E41C02915Fe1beFA78c556f946E5F20ffBD3;
-
-    uint256 constant DEC_01_2021    = 1638316800;
-    uint256 constant DEC_31_2021    = 1640908800;
-    uint256 constant JAN_01_2022    = 1640995200;
-    uint256 constant APR_30_2022    = 1651276800;
-    uint256 constant JUN_30_2022    = 1656547200;
-    uint256 constant AUG_01_2022    = 1659312000;
-    uint256 constant NOV_30_2022    = 1669766400;
-    uint256 constant DEC_31_2022    = 1672444800;
-    uint256 constant SEP_01_2024    = 1725148800;
+    address constant COM_WALLET                   = 0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9;
```

Comment:
We should move this to https://github.com/makerdao/spells-mainnet/blob/e95b9b1fd0882d890067b0620bf28c268459439c/src/test/addresses_mainnet.sol#L328-L335

and uncomment the contract so we can start keeping a list of CU addresses

Reply:
@canokaue On further thought, would you mind moving that contract to its own file? `addresses_coreunits.sol`? That will make it easier to track as we start to use it more.  Thanks!